distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -1343,9 +1364,9 @@ TLS_RSA_WITH_AES_128_CBC_SHA
 ==== Update {branding} Configuration
 
 ===== Configure {branding} Web Service Providers
-By default Solr, STS server, STS client and platform global config settings point to *localhost* and not to the fully qualified domain name of the {branding} instance. Assuming the {branding} instance is providing these servcies, the configuration must be updated to use the {branding} instance's *fully qualified domain name*  as the service provider.
+By default Solr, STS server, STS client and the rest of the system use the system property 'org.codice.ddf.system.hostname' which is defaulted to 'localhost' and not to the fully qualified domain name of the {branding} instance. Assuming the {branding} instance is providing these servcies, the configuration must be updated to use the {branding} instance's *fully qualified domain name*  as the service provider.
 
-This can be done throught the *Admin UI* application, the *Felix console*, or from the {branding} console. The most common method is to use the *Admin UI*.
+This can be done through the installer or editing the system.properties in <INSTALL_DIRECTORY>/etc/
```

Comment:
I wrote this before we talked yesterday afternoon. I will remove mention of the installer.